### ISSUE TITLE
selectable chart for volume on stakeo

### ISSUE THREAD

**Issue by borgqueenx** [blocksentinel/stakeo#12]
the chart of volume only shows 1 month volume. it would be great to also get insight into 3 year, 1 year volume etc :)

**Comment by majordutch** [comment]
Hi there! Agreed, this is needed. This addition is currently blocked by some backend changes we are working on for the API, but the plumbing exists and this feature is planned.

I will keep this open and will update here once there is movement on it.

**Comment by majordutch** [comment]
As there are now a couple built-out resources for Switcheo Ecosystem volume tracking, we have decided to reduce the scope of this utility to focus solely on the calculator aspect. We do have plans to eventually introduce our own implementation of volume and analytics tracking in a secondary web application. However, that is low priority at the moment while we engage community development for the Switcheo Ecosystem via the [Mai Te Pora](https://github.com/Mai-Te-Pora) development team.

That said, I am going to close this out. Thanks for your support!
